dump/dbus_util.cpp

```diff
@@ -138,7 +138,7 @@ bool isSystemHMCManaged(sdbusplus::bus::bus& bus)
         return false;
     }
 
-    log<level::INFO>("Util system is not HMC managed");
+    log<level::INFO>("DumpOffload system is not HMC managed");
```

Comment:
At LN128 and LN141 the log messages are only getting changed. Not any functionality. I think this can be made in a separate commit. Just a suggestion